edalize/surelog.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import os.path
+
+from edalize.edatool import Edatool
+
+logger = logging.getLogger(__name__)
+
+class Surelog(Edatool):
+
+    argtypes = ['vlogdefine', 'vlogparam']
+
+    @classmethod
+    def get_doc(cls, api_ver):
+        if api_ver == 0:
+            return {'description' : "Surelog",
+                    'lists' : [
+                        {'name' : 'library_files',
+                         'type' : 'String',
+                         'desc' : 'List of the library files for Surelog'},
+                        ]}
+
+    def configure_main(self):
+        (src_files, incdirs) = self._get_fileset_files()
+        file_list = []
+        for f in src_files:
+            if f.file_type in ['verilogSource']:
```

Comment:
This will ignore e.g. verilogSource-95 types. Use startswith instead

Reply:
Fixed.